wall_follow_node:
  ros__parameters:
    kp: 0.5
    ki: 0.0
    kd: 0.01
    dist: 0.85
    beam_a_start: -0.8475
    beam_b_start: -1.5708
    angle_range: 1.2217
    valid_reading_range: 20.0
    following_left_wall: False
